components/log_archive/policy.json.tpl

```diff
@@ -0,0 +1,21 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "PapertrailLogArchive",
+        "Effect": "Allow",
+        "Principal": {
+          "AWS": [
+            "arn:aws:iam::719734659904:root"
```

Comment:
This is Papertrail's AWS account ID, referenced from [this documentation](https://help.papertrailapp.com/kb/how-it-works/automatic-s3-archive-export/#alternative-define-sharing-policy-with-iam).